response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -284,7 +307,8 @@ def create_collection_exercise(survey_id, survey_name, user_description, period)
         "userDescription": user_description,
         "exerciseRef": period,
     }
-
+    if eq_version != "":
```

Comment:
for collection exercises which are in LIVE status in prod the eqVersion will be None and the frontstage template is checking if its None it will show the default version selection as V2 and hence it is being used this way